src/sentry/search/events/builder/metrics.py

```diff
@@ -1083,3 +1085,239 @@ def run_query(self, referrer: str, use_cache: bool = False) -> Any:
             "data": list(time_map.values()),
             "meta": [{"name": key, "type": value} for key, value in meta_dict.items()],
         }
+
+
+class TopEventsMetricQueryBuilder(TimeseriesMetricQueryBuilder):
+    def __init__(
+        self,
+        params: ParamsType,
+        interval: int,
+        dataset: Dataset,
+        top_events: List[Dict[str, Any]],
+        other: bool = False,
```

Comment:
If other with multiple series isn't supported, can we have that throw an IncompatibleMetricsQuery?

Reply:
See the top level PR comment -- but my current theory is other with multiple only doesn't work in the Metric Layer